setup.py

```diff
@@ -15,7 +15,7 @@
 
 INSTALL_REQUIRES = [
     "attrs >= 16.1.0",
-    "jsonschema >= 2.5.1",
+    "jsonschema >= 2.5.1, < 3.0.0",
```

Comment:
Please open a new issue about supporting jsonschema > 3.0.0. We should try to be as flexible as possible in low level libraries like okonomiyaki.

Reply:
I will open the issue. I think the reason here is that jsonschema 3.0.0 just came out 4 days ago and the issues are still being worked out.